bolt-modules/prompt/lib/puppet/functions/prompt.rb

```diff
@@ -11,12 +11,16 @@
   # @option options [Boolean] sensitive Disable echo back and mark the response as sensitive.
   #   The returned value will be wrapped by the `Sensitive` data type. To access the raw
   #   value, use the `unwrap` function (i.e. `$sensitive_value.unwrap`).
+  # @option options [String] default The default value to return if the user does not provide
+  #   input or if standard in (stdin) is not a tty.
```

Comment:
To be perfectly honestly I've never seen it referred to as standard in - @hestonhoffman is that actually helpful to expand, or can we just say 'stdin'?

Reply:
Technically, stdin is an abbreviation for "standard input", so best practice would be to write it out the first time and use (stdin) in parenthesis (We've used it that way here https://puppet.com/docs/bolt/latest/running_bolt_commands.html). 

On the other hand, this is a code comment, and will most likely show up in reference docs, and as you've suggested, it's a very common abbreviation in this field. I think it's okay to use the abbreviation in the code comments and stick to the best practice in our "long-form" docs. How does that sound? There are probably instances in the current docs where we're not doing this. I can root those out in a separate PR.